### ISSUE TITLE
Upon Invoking to “Click to Copy” Button The Narrator Is Not Announcing The Success Message of the button. 

### ISSUE THREAD

**Issue by EDAccessibilityTCS** [docsifyjs/docsify#1917]
### Git Hub Tags
#A11yMAS; #A11yCT; #A11ySev2; #Accessibility; #AugmentationLoopTools; #WCAG4.1.2; #TCS-E+D; #DesktopWeb;

### Repro Steps

1. Navigate to quick start link and invoke it. 
 
2. And navigate to “click to copy” button and invoke it. 
 
3. Observe the narrator announcement. 

### Actual Result

Upon invoking to “click to copy” button the narrator is not announcing the success message of the button. 

### Expected Result

While the user is invoking the “click to copy” button the narrator should announce the success message of the button. 

### Video Reference

https://user-images.githubusercontent.com/113306820/198535917-302b0a04-e3ea-4082-9d66-ef1d444e2a4a.mp4

**Comment by sy-records** [comment]
This is a plugin issue, please submit an issue to `docsify-copy-code`.

https://github.com/jperasmus/docsify-copy-code

cc @jperasmus
